onadata/apps/api/models/oauthtokenstorage.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Oauth2 token storage model class
+"""
+
+from django.db import models
+from jsonfield import JSONField
+from django.conf import settings
+
+
+class OauthStorageModel(models.Model):
```

Comment:
Is the plan for this model to be used to store different OAuth credentials? Not only google? MIght be nice to have an easy way to differentiate which provider the credentials are for i.e google, microsoft, e.t.c

Reply:
At the moment, its mainly for the google Oauth2, but any other Oauth2 provider credentials may possibly also be persisted within this model.
How would you suggest we differentiat the provider?, include a separate third text field for oauth provider? - I dont see why not